apps/mudita-center-e2e/src/page-objects/modal-backup-kompakt.page.ts

```diff
@@ -113,5 +113,37 @@ class ModalBackupKompaktPage extends OverviewPage {
   public get creatingBackupProgressBarDetails() {
     return $(`//*[@data-testid="${ProgressBarTestIds.Details}"]`)
   }
+
+  public get backupInProgressModalCancelled() {
+    return $('//*[@data-testid="modal-content-backupbackup-create"]')
```

Comment:
"The title and `data-testId` seem to be different. Are you sure that is correct?"